core/src/queue.rs

```diff
@@ -854,7 +854,7 @@ mod tests {
                 .expect("Default queue config should always build")
         });
         queue.push(tx.clone(), &wsv).unwrap();
-        wsv.transactions.insert(tx.hash());
+        wsv.transactions.insert_mut(tx.hash());
```

Comment:
Hm, I thought we will never use such mutable methods now

Reply:
We use them to perform operation on wsv in batch.